<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1075</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Open6022-QT</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnExit">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>40</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>800</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>150</y>
      <width>231</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>CH1</string>
    </property>
    <widget class="QComboBox" name="comboBoxV1div">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>104</width>
       <height>61</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>2 V/div</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 V/div</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.5 V/div</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>200mV/div</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>100mV/div</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="checkBoxCH1ON">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>41</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>ON</string>
     </property>
    </widget>
    <widget class="QDial" name="dialCursorV1">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>70</y>
      <width>231</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Horizontal</string>
    </property>
    <widget class="QComboBox" name="comboSampling">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>104</width>
       <height>26</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>100ns</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>500ns</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1us</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5us</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10us</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>50us</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>100us</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>200us</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>500us</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1ms</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5ms</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10ms</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboMemDepth">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>104</width>
       <height>26</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>1Mpts</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>512Kpts</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>128Kpts</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>32Kpts</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>281</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>CH1 Info</string>
    </property>
    <widget class="QLabel" name="lblmin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Min= </string>
     </property>
    </widget>
    <widget class="QLabel" name="lblmax">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Max= </string>
     </property>
    </widget>
    <widget class="QLabel" name="lblVpp">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Vpp=</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblminVal">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblMaxVal">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblVppVal">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblfreq">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>420</y>
      <width>231</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Trigger</string>
    </property>
    <widget class="QPushButton" name="btnGet">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>ARM</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>81</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>RISE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>FALL</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>81</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>AUTO</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NORMAL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SINGLE</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="labelTrigger">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QDial" name="dialTrigger">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>91</width>
       <height>111</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>5</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxCHSel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>CH1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CH2</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QScrollBar" name="horizontalScrollBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>801</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QDial" name="dialCursorX1">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>600</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>500</number>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QDial" name="dialCursorX2">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>600</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>500</number>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelCurX12">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>640</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>280</y>
      <width>231</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>CH2</string>
    </property>
    <widget class="QComboBox" name="comboBoxV2div">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>104</width>
       <height>61</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>2 V/div</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 V/div</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.5 V/div</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>200mV/div</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>100mV/div</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="checkBoxCH1ON_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>41</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>ON</string>
     </property>
    </widget>
    <widget class="QDial" name="dialCursorV1_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1075</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_to_file"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave_to_file">
   <property name="text">
    <string>Save to file</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
